docker/bootconfig/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM golang:1.11-alpine3.8 AS builder
+FROM golang:1.13-alpine3.10 AS builder
 WORKDIR $GOPATH/src/github.com/nats-io/nats-operator/
 RUN apk add --update git
-RUN go get -u github.com/golang/dep/cmd/dep
 COPY . .
-RUN dep ensure -v -vendor-only
+RUN go get ./...
 RUN CGO_ENABLED=0 go build -o /nats-pod-bootconfig ./cmd/bootconfig/main.go
```

Comment:
Thanks for the PR, also adding `GO111MODULE=on` here I think worked for me but would be better to change to Go 1.13 and since there it is the default

Reply:
Agreed. I figured I'd bring things up to match the ci/cd job.

Reply:
TIL: `go mod download`